surt/URLRegexTransformer.py

```diff
@@ -44,16 +44,23 @@ def stripPathSessionID(path):
     return path
 
 
-# stripQuerySessionID
+# stripQuerySessionID and Google Analytics parameters
 #_______________________________________________________________________________
 _RES_QUERY_SESSIONID = [
     re.compile(b"^(.*)(?:jsessionid=[0-9a-zA-Z]{32})(?:&(.*))?$", re.I),
     re.compile(b"^(.*)(?:phpsessid=[0-9a-zA-Z]{32})(?:&(.*))?$", re.I),
     re.compile(b"^(.*)(?:sid=[0-9a-zA-Z]{32})(?:&(.*))?$", re.I),
     re.compile(b"^(.*)(?:ASPSESSIONID[a-zA-Z]{8}=[a-zA-Z]{24})(?:&(.*))?$", re.I),
     re.compile(b"^(.*)(?:cfid=[^&]+&cftoken=[^&]+)(?:&(.*))?$", re.I),
+    re.compile(b"^(.*)(?:utm_source=[0-9a-zA-Z\.]+)(?:&(.*))?$", re.I),
+    re.compile(b"^(.*)(?:utm_campaign=[0-9a-zA-Z\.]+)(?:&(.*))?$", re.I),
+    re.compile(b"^(.*)(?:utm_term=[0-9a-zA-Z\.]+)(?:&(.*))?$", re.I),
+    re.compile(b"^(.*)(?:utm_content=[0-9a-zA-Z\.]+)(?:&(.*))?$", re.I),
+    re.compile(b"^(.*)(?:utm_medium=[0-9a-zA-Z\.]+)(?:&(.*))?$", re.I),
+    re.compile(b"^(.*)(?:utm_social=[0-9a-zA-Z\.]+)(?:&(.*))?$", re.I),
+    re.compile(b"^(.*)(?:utm_social-type=[0-9a-zA-Z\.]+)(?:&(.*))?$", re.I),
+    re.compile(b"^(.*)(?:utm_brand=[0-9a-zA-Z\.]+)(?:&(.*))?$", re.I),
```

Comment:
Below are a few non-critical comments:

* These can be combined in a single RegEx
* `[0-9a-zA-Z]` can be replaced with `\w` (thought it will include underscore as well)
* The variable name `_RES_QUERY_SESSIONID`  should be changed to something more generic now
* We should also remove `fbclid` query parameter